src/core/packages/chrome/navigation/src/components/secondary_menu/item.tsx

```diff
@@ -44,7 +44,7 @@ export const SecondaryMenuItemComponent = ({
   ...props
 }: SecondaryMenuItemProps): JSX.Element => {
   const { euiTheme } = useEuiTheme();
-  const { activeItemRef } = useContext(SecondaryMenuContext);
+  const activeItemRef = useScrollToActive<HTMLLIElement>(isHighlighted);
```

Comment:
@paulinashakirova we need to use `isCurrent` here. There's a small difference. `isHighlighted` is visual, both a parent and a secondary menu item are visibly active. `isCurrent` is actually active, e.g. by adding `aria-current="page"`.


Here, it doesn't matter that much because we're only applying the logic to the secondary menu item and there only ever will be one highlighted item. This likely works well. It's just for code clarity.